tests/test_github.py

```diff
@@ -143,6 +143,8 @@ class TestGithubIssueQuery(AbstractServiceTest, ServiceTest):
         'github.password': 'arbitrary_password',
         'github.username': 'arbitrary_username',
         'github.query': 'is:open reviewer:octocat',
+        'github.include_user_repos': 'False',
+        'github.include_user_issues': 'False',
```

Comment:
Did you see my latest comment in [452](https://github.com/ralphbean/bugwarrior/pull/452)?

> The changes I made to the test configuration explain how to achieve the behaviour you want. I can understand if you wouldn't be thrilled about having to set those flags to False, but to me that's a better trade off than having the query option dynamically change the default values to False, which is what this PR currently effectively does. I might be receptive to changing those default values to False, but dynamic default values seem too confusing.